src/arch/runtime/event_queue/iocp.hpp

```diff
@@ -1,12 +1,15 @@
 #ifndef ARCH_RUNTIME_EVENT_QUEUE_IOCP_HPP_
 #define ARCH_RUNTIME_EVENT_QUEUE_IOCP_HPP_
 
+#include <stdint.h>
```

Comment:
Super, if that's correct then you should open a pull request for that change. It's not relevant to this pull though -- it doesn't make sense to change in just one place.